charactersheet/charactersheet/services/persistence_service.js

```diff
@@ -131,6 +131,42 @@ PersistenceService.findBy = function(model, property, value) {
 };
 
 /**
+ * Given a model class, return all of the stored instances that match a given
+ * array of filter criteria.
+ *
+ * Parameters
+ * ----------
+ *
+ * model: The prototype for the type of model that is being searched for.
+ * properties: A static array of properties that the data object will be 
+ * compared against.
+ *
+ * Returns
+ * -------
+ *
+ * A list of objects of the desired type, which match the list of criteria.
+ *
+ * Usage
+ * -----
+ * ```javascript
+ *         var status = PersistenceService.findByListOfProperties(Status,
+ *           [{'name': 'characterId', 'value': key},
```

Comment:
I think this is fine. It's basically a match object. If we want we could create a custom 'Predicate' object that for now would just have a key/value option. Then we could expand later.

```js
function PersistenceServicePredicate() {
  var self = this;
  self.name = ''
  self.value = ''
}

PersistenceServicePredicate.keyValuePredicate = function(key, value) {
  // ...
}
```

The only benefit I see to this is if we later add things like Regex matching, or constraints, or something.